src/processors/processVrmRequest.ts

```diff
@@ -13,8 +13,8 @@ export const processPatchVrmRequest = (currentRecord: TechRecordGet, userDetails
     const updatedNewRecord = setCreatedAuditDetails(newRecord, userDetails.username, userDetails.msOid, new Date().toISOString(), currentRecord.techRecord_statusCode as StatusCode);
     (updatedNewRecord as TechRecordHgv | TechRecordMotorcycle | TechRecordCar | TechRecordPsv).primaryVrm = newVrm.toUpperCase();
     const updatedRecordToArchive = setLastUpdatedAuditDetails(recordToArchive, userDetails.username, userDetails.msOid, new Date().toISOString(), StatusCode.ARCHIVED);
-
-    return [updatedRecordToArchive, newRecord];
+    updatedNewRecord.techRecord_reasonForCreation = 'Update VRM - Cherished Transfer';
```

Comment:
do we not send a reason for creation from the frontend? If we do, what is the point of asking the user to fill it in if we're just overwriting it here anyways?

Reply:
Neverming we don't do that

Reply:
Change VRM doesn't have an option to add reason